tests/test_stdiomgr_base.py

```diff
@@ -549,6 +550,24 @@ def test_tee_type():
     assert str(err.value) == "tee must be a TextIOBase."
 
 
+def test_non_closing_type():
+    """Test that incorrect type doesnt raise exceptions."""
+    # Ensure the type used has no __exit__ or close()
+    assert not hasattr("", "__exit__")
```

Comment:
Probably should create a variable for this string-as-type, instead of using the magic empty string throughout.